.gitignore

```diff
@@ -6,7 +6,9 @@ logs/
 # Files to ignore
 ._*
 .cache
+.config
 .DS_Store
+.env
```

Comment:
This is a file used defaultly by Docker Compose. This is environment-specific and should be ignored.

The `.config` entry was a file I kept getting created by some npm process on review instances. Was just a file containing the last update check.